recipe/symfony-flex.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Deployer;
+
+require_once __DIR__ . '/symfony3.php';
+
+/**
+ * Symfony Flex Configuration
+ */
+
+//No need to clear anything
+set('clear_paths', []);
+
+// Symfony shared dirs
+set('shared_dirs', ['var/log', 'var/sessions']);
+
+// Symfony writable dirs
+set('writable_dirs', ['var/cache', 'var/log', 'var/sessions']);
+
+//File for DotEnv if not using env vars
+set('shared_files', ['.env']);
+
+// Symfony web dir
+set('web_dir', 'public');
+
+// Assets
+set('assets', ['public/css', 'public/images', 'public/js']);
+
+// Environment vars
+set('env', function () {
+    return [
+        'APP_ENV' => get('symfony_env')
```

Comment:
We should probably add documentation to tell the user to put "symfony_env" in it's host.yml

Reply:
Before it was `SYMFONY_ENV` https://github.com/deployphp/deployer/blob/master/recipe/symfony.php#L41
But it not for flex

Reply:
Maybe:
```php
// Environment vars
set('env', function () {
    return [
        'SYMFONY_ENV' => get('symfony_env'),
        'APP_ENV'     => get('symfony_env'),
    ];
});
```

Reply:
I can not set environment variables on my server so `Dotenv` is also included in the _production_ environment. There were complications. I noticed that setting `APP_ENV` disables reading the _.env_ file, therefore I set only `symfony_env` to have all commands with `--env = {{symfony_env}}` option

Reply:
Hmm. Maybe you are right and your solution is much better than my hacky workaround https://github.com/deployphp/deployer/pull/1440#discussion_r159818610
AFAIR composer does not account `--env=`
Will test, and change it if all works fine.